src/operation/operation_problem_templates.jl

```diff
@@ -27,9 +27,11 @@ end
 
 function _default_services()
     return [
-        ServiceModel(PSY.VariableReserve{PSY.ReserveUp}, RangeReserve),
-        ServiceModel(PSY.VariableReserve{PSY.ReserveDown}, RangeReserve),
-    ]
+    # TODO: Since service models now requires service name, 
```

Comment:
Since the changes we discussed @jd-lara , it is require to add a name for each service and pre-defining a template for services becomes tricky